### ISSUE TITLE
Get-IshUser fails with 'Item has already been added. Key in dictionary: 'None' Key being added: 'None'' when running in parallel in StringEnum::GetStringValue

### ISSUE THREAD

**Issue by jlaridon** [RWS/ISHRemote#134]
When running following script multiple times in parallel

```powershell
$user = Get-IshUser -IshSession $session 
$userName = Get-IshMetadataField -IshObject $user -Name \"USERNAME\" 
return $userName"`
```
following error can occur

```powershell
Message: "Item has already been added. Key in dictionary: 'None' Key being added: 'None'"
StackTrace:
at System.Collections.Hashtable.Insert(Object key, Object nvalue, Boolean add)\r\n
at System.Collections.Hashtable.Add(Object key, Object value)\r\n
at Trisoft.ISHRemote.HelperClasses.StringEnum.GetStringValue(Enum value)\r\n
at Trisoft.ISHRemote.Objects.Public.IshField.GetXml(XmlWriter& xmlWriter)\r\n
at Trisoft.ISHRemote.Objects.IshFields.GetXml(XmlWriter& xmlWriter)\r\n
at Trisoft.ISHRemote.Objects.IshFields.ToXml()\r\n
at Trisoft.ISHRemote.Cmdlets.User.GetIshUser.ProcessRecord()
```

**Comment by ddemeyer** [comment]
Typically PowerShell is not multi-threaded unless you force it over Runspaces. In turn most of ISHRemote is thread-safe as cmdlets retrieve all their parameters from _the outside_. However, IshSession creation which is lean in the beginning, but as you ask/use it more it will keep a cache copy of the retrieved information. This issue is one of those, there is a static hashtable to speed up enum-conversion that happens numerously per cmdlet/API-call. The hashtable is created as we go by adding to it through [StringEnum::GetStringValue](https://github.com/RWS/ISHRemote/blob/master/Source/ISHRemote/Trisoft.ISHRemote/HelperClasses/StringEnum.cs#L162) 

So either remove the (theoretical speedup) by removing the hashtable. Or make the hashtable threadsafe.

**Comment by ddemeyer** [comment]
Closes #134
